tests/test_validation.sh

```
@@ -0,0 +1,85 @@
+    "jfranklin-foundry/task-1-microsoft-Phi-3-mini-4k-instruct"
+    "jack8885/task-1-microsoft-Phi-3.5-mini-instruct"
+    "nannnzk/task-1-Qwen-Qwen2.5-7B-Instruct"
+    "Howell2333/task-1-google-gemma-7b"
+    "jake-dcg/task-2-google-gemma-2-2b-it"
+)
+
+# Run validation for each model
+for MODEL in "${MODELS[@]}"; do
+    echo "Validating model: $MODEL"
+
+    cd src
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)